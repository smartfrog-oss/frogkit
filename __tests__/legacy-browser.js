import { mount } from 'vue-test-utils'
import LegacyBrowser from '@/components/legacy-browser'

describe('Legacy Browser', () => {
  it('should render component and match snapshot', () => {
    const wrapper = mount(LegacyBrowser)
    expect(wrapper.html()).toMatchSnapshot()
  })
  it('should not emit click event', () => {
    const stub = jest.fn()
    const wrapper = mount(LegacyBrowser)
      wrapper.vm.$on('click', stub)
      wrapper.vm.$el.querySelector('.fk-legacy-browser').click()
      expect(stub).not.toBeCalled()
  })
})
