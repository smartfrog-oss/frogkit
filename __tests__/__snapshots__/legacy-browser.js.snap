// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Legacy Browser should render component and match snapshot 1`] = `
<darklayer>
  <flex justify="space-around" align="center" column="" class="fk-legacy-browser">
    <div class="m-b-10">
      <p class="fk-legacy-browser__title"></p>
      <p></p>
    </div> <img src="../assets/images/chrome.svg" class="m-b-10"> <button link="" color="secondary" target="_blank" href="https://www.google.com/chrome/browser/desktop/"></button>
  </flex>
</darklayer>
`;
